esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4563,6 +4573,9 @@ void CWsWuFileHelper::createZAPWUQueryAssociatedFiles(IConstWorkUnit* cwu, const
         cur.getName(name);
         cur.getIp(ip);
 
+        if (!hasLogsAccess && endsWith(name.str(), ".eclcc.log"))
```

Comment:
I'm not sure I follow what's special about  *.eclcc.log files, this might need a code comment

Reply:
The eclcc log files are included as "Associated Files" for bare-metal clusters. I'm checking for them specifically here to exclude if the user doesn't have access. I considered checking for any *.log file but thought it might be best to limit the exclusion to just the log file I'd seen could be included here.

Reply:
Thanks @asselitx, as mentioned offline, eclcc.log files aren't typically creating via the logAccess feature, and the logAccess feature level flag might not apply here.